brightcove-exoplayer/AdRulesIMAWidevineModularSampleApp/README.md

```diff
@@ -0,0 +1 @@
+Placeholder
```

Comment:
As a general practice, since this file is displayed automagically on GitHub, we should probably add some useful text in our README.md files.